war/src/main/scss/modules/dialogs.scss

```diff
@@ -0,0 +1,83 @@
+$jenkins-dialog-padding: 1rem;
+
+.jenkins-dialog {
+  border-radius: 0.6rem;
+  border: none;
+  background: transparent;
+  box-shadow: var(--modal-box-shadow);
+  backdrop-filter: var(--modal-backdrop-filter);
+  animation: jenkins-dialog-animate-in 0.25s cubic-bezier(0, 0.68, 0.5, 1.5);
+  overflow: hidden;
+  padding: 0;
+
+  &::backdrop {
+    background: var(--modal-backdrop-background);
+    animation: jenkins-dialog-backdrop-animate-in 0.15s;
+  }
+
+  &__contents {
+    overflow-y: auto;
+    overflow-wrap: break-word;
+    padding: $jenkins-dialog-padding;
+    max-height: 75vh;
+
+    &--title {
+      padding-top: 0;
+    }
+  }
+
+  &__input {
+    display: flex;
+    padding: 0 1rem 1rem;
+  }
+
+  &__buttons {
+    padding: 0.5rem 1rem 1rem;
+    justify-content: center;
+  }
+
+  &__title {
+    font-size: 1.1rem;
+    font-weight: 600;
+    padding: $jenkins-dialog-padding;
+  }
+
+  &__subtitle {
+    font-size: 1rem;
+    font-weight: 600;
+    color: var(--text-color-secondary);
+    padding: 0;
+    margin: 0 0 1rem;
+  }
+
+  &__close-button {
+    position: absolute;
+    top: $jenkins-dialog-padding - 0.5rem;
+    right: $jenkins-dialog-padding - 0.5rem;
+    aspect-ratio: 1;
+    padding: 0;
+
+    &::before,
+    &::after {
+      border-radius: 100%;
+      outline: none;
+    }
+
+    &::after {
+      backdrop-filter: contrast(0.9) blur(5px);
+    }
+  }
+}
+
+@keyframes jenkins-dialog-backdrop-animate-in {
```

Comment:
There are currently no animate-out transitions. When I add them then htmlunit tests are not properly performing the page submit after clicking the ok button. On a live system everything works fine.